stable/elasticsearch-curator/values.yaml

```diff
@@ -103,3 +103,38 @@ priorityClassName: ""
 
 securityContext:
   runAsUser: 16  # run as cron user instead of root
+
+# Add your own init container or uncomment and modify the given example.
+initContainers: {}
```

Comment:
by convention, we call this `extraInitContainers`

Reply:
Thanks for the recommendation. Updated the pull request with this change.